optimum/neuron/accelerate/accelerator.py

```diff
@@ -422,7 +422,8 @@ def _prepare_model_for_mp(
         cpu_ids = {name: id(param) for name, param in model.named_parameters()}
         tied_parameters_dict = get_tied_parameters_dict(model)
         model_main_input_name = getattr(model, "main_input_name", None)
-        model = self.state.mp_plugin.parallelize_model(model, device=self.device)
+        # TODO: use self.device.
```

Comment:
You changed that back also ... Is it because the tests were failing ?

Reply:
Yes, and since it is not related to the PR, I will work on that on another PR.